includes/polling/os/draytek.inc.php

```diff
@@ -3,7 +3,7 @@
  * draytek.inc.php
  * @author     Jason Cheng <sanyu3u@gmail.com>
  */
-preg_match('/Router Model: ([\w ]+), Version: ([\w\.]+)/', $poll_device['sysDescr'], $tmp_draytek);
```

Comment:
If you use `$device['sysDescr']` does it work?  It would be really helpful if you provided the sysDescr from the device.